cmake/oneflow.cmake

```
@@ -290,6 +290,17 @@ set_property(TARGET LLVMSupportWithHeader PROPERTY INTERFACE_INCLUDE_DIRECTORIES
                                                    ${LLVM_INCLUDE_DIRS})
 
 list(APPEND oneflow_third_party_libs LLVMSupportWithHeader)
+find_package(BFD)
+if(BFD_FOUND)
+  add_definitions(-DBACKWARD_HAS_BFD)
+  list(APPEND oneflow_third_party_libs ${BFD_LIBRARIES})
+  target_include_directories(oneflow ${BFD_INCLUDE_PATH})
+endif()
+find_package(Unwind)
+if(Unwind_FOUND)
+  add_definitions(-DBACKWARD_HAS_LIBUNWIND)
+  list(APPEND oneflow_third_party_libs ${Unwind_LIBRARY})
```

现代写法：
```suggestion
  list(APPEND oneflow_third_party_libs unwind::unwind)
```
unwind::unwind 是个 target（也是在 FindUnwind.cmake 里定义的），它同时包含了 lib 和 include

FindBFD.cmake 也可以升级一下，仿照 FindUnwind.cmake 定义一个叫 bfd::bfd 的 target